src/scripts/eos-analysis

```diff
@@ -629,7 +629,7 @@ def main():
 
             args.cmd(args)
     except Exception as e:
-        print(colored('✖ Encountered an unrecoverable error', 'red', attrs=['bold']), f'{e}')
+        print(colored('✖ Encountered an unrecoverable error:', 'red', attrs=['bold']), f'{e}')
```

Comment:
Should we add a line break here?

Reply:
I think it's fine as is, but I don't have a strong opinion